CMakeLists.txt

```diff
@@ -255,6 +255,10 @@ if(ENABLE_GOLDMASTER)
 	add_definitions(-DENABLE_GOLDMASTER)
 endif()
 
+# Qt defines 'emit' as macros, which conflicts with TBB definition of method with same name
+add_definitions(-DQT_NO_EMIT)
```

Comment:
it should be added to launcher's CML

Reply:
this workaround is also good:
```
#ifndef Q_MOC_RUN
#include <tbb/tbb.h>
#endif
```

Reply:
Moved to map editor CMakeList.

Still workaround, but should be more reliable than correct include order which can easily break in future if we were to use something else from tbb.